pkgs/by-name/gi/gitea-mcp-server/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitea,
+}:
+buildGoModule rec {
```

rec to point to versions is deprecated, you shouldn't use it here.
```suggestion
buildGoModule (finalAttrs: {
```